library/std/src/thread/scoped.rs

```diff
@@ -136,14 +136,12 @@ pub fn scope<'env, F, T>(f: F) -> T
 where
     F: for<'scope> FnOnce(&'scope Scope<'scope, 'env>) -> T,
 {
-    // We put the `ScopeData` into an `Arc` so that other threads can finish their
```

Comment:
that justification seems fairly clear, so you'd need to show why it's incorrect

Reply:
Idk, it wasn't clear to me when I read it. The way I saw it, the scoped data outlives the place where it is joined, so it seemed reasonable a reference should outlive it.

Reply:
improving the comment would be a nice improvement then :)

Reply:
see https://github.com/rust-lang/rust/issues/116237

Reply:
> improving the comment would be a nice improvement then :)

Yeah, definitely.